app/helpers/dashboard_helper.rb

```diff
@@ -90,4 +90,18 @@ def report_color
     }
   end
 
+  def auto_refresh_button(options = {})
+    on = options[:defaults_to] ? "on" : "off"
+    if params[:auto].present?
```

Comment:
nitpick: can we call it something like auto_refresh or refresh=true instead? plain auto looks odd?